10up/distributor
# Prevent distribution of `classic-editor-remember` post meta.

### Description of the Change


Prevents the distribution of the `classic-editor-remember` post meta data from one site to another.

This setting is used to determine whether a post should be opened in the block or classic editor on sites with the classic editor plugin installed.

On sites with the plugin installed, the preference should be made by that sites editors. On sites without the plugin installed then the meta data is not needed and fills their post meta table with junk.


Closes #976 

### How to test the Change


1. Install the classic editor on two Distributor enabled site
2. Set it to allow users to switch between the editor of their choice (Settings > Writing)
3. Author and publish a post in the block editor
4. Open the post in the classic editor
5. Close without saving.
6. Either push or pull the post to another site
7. Edit the Distributed post on the second site
8. Ensure the post still opens in the block editor 

### Changelog Entry

> Fixed - Prevent distribution of classic editor plugin related meta data.


### Credits

Props @peterwilsoncc, @faisal-alvi, @jeffpaul 


### Checklist:
